source/extensions/common/wasm/wasm.cc

```diff
@@ -71,6 +69,20 @@ std::string Sha256(absl::string_view data) {
 
 namespace {
 
+#define CREATE_WASM_STATS(COUNTER, GAUGE)                                                          \
+  COUNTER(remote_load_cache_hits)                                                                  \
+  COUNTER(remote_load_cache_negative_hits)                                                         \
+  COUNTER(remote_load_cache_misses)                                                                \
+  COUNTER(remote_load_fetches)                                                                     \
+  COUNTER(remote_load_fetch_successes)                                                             \
+  COUNTER(remote_load_fetch_failures)                                                              \
```

Comment:
Isn't `remote_load_fetches` effectively an "eventually consistent sum" of `remote_load_fetch_successes` and `remote_load_fetch_failures`, therefore redundant?

Reply:
Yes.  However, you can use the eventual part to figure out if anything is in flight.   Do you want me to remove it?

Reply:
I don't particularly care, to be honest. @mandarjog?

Reply:
@PiotrSikora good point. We should remove redundant metric.

Reply:
Done.